radxa.c

```diff
@@ -312,9 +314,37 @@ static int changeOwner(char *file) {
 	return 0;
 }
 
+static int radxaKernelRev(void) {
+	FILE *versionFd;
+	char line[120];
+
+	if((versionFd = fopen("/proc/version", "r")) == NULL) {
+		wiringXLog(LOG_ERR, "radxa->identify: Unable open /proc/version");
+		return -1;
+	}
+
+	fgets(line, 120, versionFd);
+	fclose(versionFd);
+
+	if(strstr(line, "3.0.36") != NULL) {
```

Comment:
Are we sure there is only one 3.0.36 version and one 3.18.0 version? Maybe we can better check for versions below 3.18.0?